src/pages/HomePage/components/Rating/Rating.tsx

```diff
@@ -0,0 +1,33 @@
+interface IRatingProps {
+    zeroData: string;
+    firstData: string;
+    secondData: string;
+    thirdData: string;
+    forthData: string;
+    fifthData: string;
+    star: string;
+  }
+  
+  export const Rating = (props: IRatingProps) => {
+    const { zeroData, firstData, secondData, thirdData, forthData, fifthData, star } = props;
+    return (
+      <div className="rating" data-total-value={zeroData}>
+        <div className="stars" data-item-value={fifthData}>
+          {star}
+        </div>
+        <div className="stars" data-item-value={forthData}>
+          {star}
+        </div>
+        <div className="stars" data-item-value={thirdData}>
+          {star}
+        </div>
+        <div className="stars" data-item-value={secondData}>
+          {star}
+        </div>
+        <div className="stars" data-item-value={firstData}>
+          {star}
+        </div>
+      </div>
+    );
+  };
```

Comment:
nice try =)
you see, you need to send a number (the rating's value) to this component, and this component must decide how many yellow stars and how mane gray stars will be))